docs/aws/ContinuousIntegration/WebApp.rst

```diff
@@ -0,0 +1,105 @@
+.. _aws-continuous-integration-web-app:
+
+Continuous integration of the AWS web application
+#################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+
+    This is an advanced topic that is closely tied with the further development and customization of the nRF Asset Tracker for your purposes.
+    See the `GitHub project page of the nRF Asset Tracker Web Application for AWS <https://github.com/NordicSemiconductor/asset-tracker-cloud-app-aws-js/>`_ for an implementation of the process outlined in this section.
+
+Every change to the project is tested using a head-less browser with the `Playwright <https://Playwright.dev>`_ framework. This ensure that all dependencies of the web application work as intended and present the data in the correct way.
+
+Running end-to-end tests during development
+*******************************************
+
+The end-to-end tests run against an instance of the :ref:`nRF Asset Tracker for AWS <index_aws>`.
+
+Either, use the credentials you created, :ref:`when setting up the solution <aws-getting-started>`, or enable the Web App CI feature and use the dedicated credentials created for this task.
+The latter option is the recommended approach since it limits the permission scope to only the needed ones. 
+They can also be used to :ref:`run the end-to-end tests on GitHub Actions <aws-continuous-integration-web-app-github>`.
+
+Add these environment variables to your ``.envrc`` file:
+
+.. parsed-literal::
+    :class: highlight
+
+    # add to .envrc
+    export AWS_REGION="*your preferred AWS region*"
+    export AWS_ACCESS_KEY_ID="*value of Access key ID*"
+    export AWS_SECRET_ACCESS_KEY="*value of Secret access key*"
+    export WEBAPP_STACK_NAME="*name of your web application stack*"
+
+Run the following command to allow the changed ``.envrc`` file
+
+.. code-block:: bash
+
+  direnv allow
+
+Running the tests
+-----------------
+
+You can then run the tests using
+
+.. code-block:: bash
+
+  npx playwright test
```

Comment:
A subheading for one command seems extraneous, especially since it's already under a heading about running the tests.